Select Allowed Loadouts in Round Settings

mes258 opened the pull request:
Host can select which loadouts are permitted in each round.

mes258 committed:
Added loadout select to map select

mes258 committed:
Store loadouts as numbers, not strings

mes258 committed:
Loadouts switch at new round start

mes258 committed:
host can change round settings between rounds

mes258 committed:
updated round settings UI

vinport16 reviewed (changes requested):
<img width="831" alt="Screen Shot 2023-01-28 at 7 48 38 AM" src="https://user-images.githubusercontent.com/3358441/215267248-96d37431-8ff4-410b-8474-0fc931069afc.png">
the leaderboard blocks me from using the buttons. I think it will be easy to push the leaderboard behind the settings though..
otherwise I think this makes sense..

vinport16 commented:
also wondering why you moved the round settings to the right?

mes258 committed:
Fixed leaderboard blocking issue

mes258 commented:
Fixed the leader board issue, thanks. I moved the round settings to the left so the instructions were centered on the whole screen. Also, now the host moves across this screen left to right. First they set up the round, then read the instructions and click play.
